edit-post/api/sidebar.js

```diff
@@ -93,6 +96,5 @@ export function getSidebarSettings( name ) {
  * @return {void}
  */
 export function activateSidebar( name ) {
```

Comment:
@omarreiss @Xyfi @IreneStr, can we get rid of this function completely and start using `dispatch( 'edit-post' ).openGeneralSidebar( name );` directly?

Reply:
Question 2, should we rename `openGeneralSidebar` to `openEditorSidebar` and use the name editor sidebar in other places?

Reply:
I guess it would be fine to get rid of this function. I still see the use for a higher level API like this because it simply requires less knowledge of the user. But maybe it is a bit premature to be adding API's like this one at this stage. I would like to make a knowledge matrix further down the road to map what knowledge is needed to solve a particular kind of problem in Gutenberg as a plugin author. Based on that we could always decide to reintroduce helpers like this one.

Reply:
I will open a separate PR for that together with docs update. This PR got big once I started integrating `data` module. We can wrap it later back with something that you will find out helpful when integrating plugins.